sema/commons/glob/globbery.py

```
@@ -0,0 +1,88 @@
+from abc import ABC, abstractmethod
+from pathlib import Path
+from typing import Any, Dict, List
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)